src/SprykerEco/Yves/Payone/Form/OnlineTransferSubForm.php

```diff
@@ -52,21 +51,29 @@ public function getTemplatePath()
     }
 
     /**
-     * @param \Symfony\Component\OptionsResolver\OptionsResolverInterface $resolver
+     * @param \Symfony\Component\OptionsResolver\OptionsResolver $resolver
      *
      * @return void
      */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function configureOptions(OptionsResolver $resolver)
     {
-        parent::setDefaultOptions($resolver);
-
         $resolver->setDefaults([
             'data_class' => PayonePaymentOnlinetransferTransfer::class,
             'constraints' => [
                 // Add Callback constraint for bank account check in ancestor classes
```

Comment:
What about this part?

Reply:
As I understood it came from core6 have no idea what to do here.